google-map.html

```diff
@@ -431,7 +441,7 @@
         return;
       }
       this._mutationObserver = new MutationObserver( this._updateMarkers.bind(this));
-      this._mutationObserver.observe(this, {
+      this._mutationObserver.observe(this.$.selector, {
```

Comment:
Try `Polymer.dom(this.$.selector)`. If that doesn't work, try putting the MO on `Polymer.dom(this)`. The latter will observe light dom children.